settings/src/style.scss

```diff
@@ -5,6 +5,10 @@
 	margin-bottom: 2rem;
 }
 
+a[data-content*="notice-info-link"] {
+	text-decoration: underline;
+}
+
```

Comment:
Since the other two don't have `<a>` on them, and I assumed this issue would only happen with the "info" option (where the contrast ratio of `text : background color` and `text : surrounding text color` are not sufficient), I didn't specifically think about handling it. But it makes sense to apply this to all `.notice` links I believe 👍

Reply:
Fixed in https://github.com/WordPress/wporg-two-factor/pull/125/commits/1ef8591a093b3110d5dbb54d274bc55d1b003122